proto/config.proto

```diff
@@ -1,18 +1,22 @@
+syntax = "proto3";
+
 package oci;
 
+import "google/protobuf/any.proto";
+
 // Spec is the base configuration for the container. It specifies platform
 // independent configuration.
 message Spec {
 	// Version is the version of the specification that is supported.
-	optional string version = 1;
+	string version = 1;
 	// Platform is the host information for OS and Arch.
-	optional Platform platform = 2;
+	Platform platform = 2;
 	// Process is the container's main process.
-	optional Process process = 3;
+	Process process = 3;
 	// Root is the root information for the container's filesystem.
-	optional Root root = 4;
+	Root root = 4;
 	// Hostname is the container's host name.
-	optional string hostname = 5;
+	string hostname = 5;
 	// Mounts profile configuration for adding mounts to the container's
 	// filesystem.
 	repeated MountPoint mounts = 6;
```

Comment:
I'm not sure if google is moving away from this, but the general protobuf standard has been to not use plural names for repeated fields, so this would be mount or mount_point.